workers/unity/Packages/com.improbable.gdk.buildsystem/Util/PathUtils.cs

```diff
@@ -0,0 +1,33 @@
+using System.IO;
+using System.Linq;
+
+namespace Improbable.Gdk.BuildSystem
+{
+    /// <summary>
+    ///     Contains common directories related to building assets and players.
+    /// </summary>
+    /// <remarks>
+    ///     All directories should be in Unity path format e.g. "Foo/Bar".
+    /// </remarks>
+    public static class PathUtils
+    {
+        public static readonly string BuildScratchDirectory = Path.GetFullPath(Path.Combine("build", "worker"));
```

Comment:
These aren't really very "Utility", and could be folded somewhere else.

Reply:
I agree, previously they were in `EditorPaths`, feels like they still belong there